pic-sure-api-war/src/main/java/edu/harvard/dbmi/avillach/service/PicsureInfoService.java

```diff
@@ -53,9 +54,10 @@ public ResourceInfo info(UUID resourceId, QueryRequest credentialsQueryRequest)
 	 *
 	 * @return List containing limited metadata about all available resources and ids.
 	 */
-	public List<Resource> resources() {
+	public List<UUID> resources() {
 		//TODO Need to limit the metadata returned
-		return resourceRepo.list();
+		// DONE: nbenik 2019-08-23
+		return resourceRepo.list().stream().map(Resource::getUuid).collect(Collectors.toList());
 	}
```

Comment:
If this returns a List UUID, the user wouldn't be able to see the resource details, they have to do another request, probably GET, to grab the resource detail.

Reply:
This is the intended original functionality. The user may not always be allowed to see the details of all resources, which we could not control before.